packages/app/src/systems/Network/components/NetworkItem/NetworkItem.tsx

```diff
@@ -37,7 +37,7 @@ export const NetworkItem = forwardRef<HTMLDivElement, NetworkItemProps>(
             variant="link"
             icon={<Icon icon={Icon.is('Pencil')} />}
             aria-label="Update"
-            onPress={() => onUpdate?.(network)}
+            onPress={() => onUpdate?.(network.id as string)}
```

Comment:
enforcing a type to match function input typing is basically throw away typescript benefits and open a door to break the code

instead we can change `onUpdate` function to accept undefined